packages/data-stores/src/queries/use-launchpad.ts

```diff
@@ -69,14 +74,31 @@ export const fetchLaunchpad = (
 		  } as APIFetchOptions );
 };
 
+const addOrderToTask = ( task: Task, index: number ) => {
+	task.order = index;
+	return task;
+};
+
+export function sortLaunchpadTasksByCompletionStatus( response: LaunchpadResponse ) {
+	const tasks = response.checklist || [];
+	const completedTasks = tasks.filter( ( task: Task ) => task.completed );
+	const incompleteTasks = tasks.filter( ( task: Task ) => ! task.completed );
+	response.checklist = [ ...completedTasks, ...incompleteTasks ].map( addOrderToTask );
+	return response;
+}
+
 export const useLaunchpad = (
 	siteSlug: string | null,
-	checklist_slug?: string | 0 | null | undefined
+	checklist_slug?: string | 0 | null | undefined,
+	options?: UseLaunchpadOptions
 ) => {
 	const key = [ 'launchpad', siteSlug, checklist_slug ];
+	const defaultSuccessCallback = ( response: LaunchpadResponse ) => response;
```

Comment:
Nit: we could move this definition out of `useLaunchpad`, as it's very much a standalone function like `addOrderToTask` and `sortLaunchpadTasksByCompletionStatus`. It doesn't need to be exported, but can simply be moved out of this function.

Reply:
Also, should the default not call `addOrderToTask()` so we get order-related data by default?
```suggestion
	const defaultSuccessCallback = ( response: LaunchpadResponse ) => {
		const tasks = response.checklist || [];
		response.checklist = tasks.map( addOrderToTask );
		return response;
	}
```